internal/controller/controllers/controller_event_wrapper.go

```diff
@@ -90,3 +102,13 @@ func (c *controllerEventsWrapper) NotifyKubeApiHostEvent(clusterID strfmt.UUID,
 		c.NotifyKubeApiClusterEvent(*host.ClusterID)
 	}
 }
+
+func (c *controllerEventsWrapper) NotifyKubeApiInfraEnvEvent(infraEnvId strfmt.UUID) {
```

Comment:
Is there a need to add a call to `NotifyKubeApiInfraEnvEvent` from  `NotifyKubeApiHostEvent` ?

Reply:
@filanov?

Reply:
No need